fern/images/logo-white.svg

```diff
@@ -1,4 +1,4 @@
-<path d="M356.495 236.668C333.276 217.039 298.297 209.171 267.295 232.085C265.868 233.122 264.095 231.35 265.176 229.966C272.527 220.498 281.045 210.295 287.92 200.048C294.924 189.543 305.388 182.02 317.451 178.345C381.66 158.89 362.376 64.0342 362.376 64.0342C362.376 64.0342 263.187 70.4328 275.424 155.993C277.456 170.303 273.651 184.873 264.701 196.244C253.718 210.122 240.963 223.395 231.71 232.993C229.764 234.981 226.478 233.079 227.256 230.399C236.207 200.265 242.736 153.658 211.734 123.611L168.106 87.3805L159.718 98.4484C134.77 131.35 142.077 177.74 175.024 202.642C193.92 216.91 202.481 232.431 201.14 249.465C200.319 259.668 195.692 269.223 188.774 276.789C175.76 291.056 163.61 306.361 154.227 324.087C152.93 326.551 149.168 325.6 149.298 322.79C150.638 293.52 147.828 227.545 98.536 203.983L43.364 182.668L39.0834 195.422C25.204 236.581 47.904 280.55 89.0236 294.515C124.782 306.663 137.537 329.707 128.933 364.251C128.543 365.505 122.317 401.086 123.182 416.867H162.831C164.172 392.396 189.855 376.313 212.123 386.3C218.392 389.111 224.835 393.131 231.45 398.32C266.906 426.249 319.138 419.634 347.026 384.139L354.982 374.022L304.826 338.008C270.408 310.944 224.489 323.179 190.504 346.352C187.65 348.298 184.018 345.185 185.618 342.072C226.694 261.484 280.093 261.657 301.021 279.556C326.402 301.259 364.84 297.368 386.373 271.903L392.556 264.597L356.452 236.668H356.495Z" fill="#FBFFFA"/>
+<path d="M356.495 236.668C333.276 217.039 298.297 209.171 267.295 232.085C265.868 233.122 264.095 231.35 265.176 229.966C272.527 220.498 281.045 210.295 287.92 200.048C294.924 189.543 305.388 182.02 317.451 178.345C381.66 158.89 362.376 64.0342 362.376 64.0342C362.376 64.0342 263.187 70.4328 275.424 155.993C277.456 170.303 273.651 184.873 264.701 196.244C253.718 210.122 240.963 223.395 231.71 232.993C229.764 234.981 226.478 233.079 227.256 230.399C236.207 200.265 242.736 153.658 211.734 123.611L168.106 87.3805L159.718 98.4484C134.77 131.35 142.077 177.74 175.024 202.642C193.92 216.91 202.481 232.431 201.14 249.465C200.319 259.668 195.692 269.223 188.774 276.789C175.76 291.056 163.61 306.361 154.227 324.087C152.93 326.551 149.168 325.6 149.298 322.79C150.638 293.52 147.828 227.545 98.536 203.983L43.364 182.668L39.0834 195.422C25.204 236.581 47.904 280.55 89.0236 294.515C124.782 306.663 137.537 329.707 128.933 364.251C128.543 365.505 122.317 401.086 123.182 416.867H162.831C164.172 392.396 189.855 376.313 212.123 386.3C218.392 389.111 224.835 393.131 231.45 398.32C266.906 426.249 319.138 419.634 347.026 384.139L354.982 374.022L304.826 338.008C270.408 310.944 224.489 323.179 190.504 346.352C187.65 348.298 184.018 345.185 185.618 342.072C226.694 261.484 280.093 261.657 301.021 279.556C326.402 301.259 364.84 297.368 386.373 271.903L392.556 264.597L356.452 236.668H356.495Z" fill="#1DA32B"/>
```

Comment:
fysa this is also used here so not sure if this is kosher to change https://github.com/fern-api/fern/blob/4ac9755776a81e8325361b189ac8c53b748cc0b8/fern/docs.yml#L581